### ISSUE TITLE
portable mode on windows no longer works with atom 1.14.0 beta

### ISSUE THREAD

**Issue by sagiegurari** [atom/atom#13601]
### Description

portable mode on windows no longer works with atom 1.14.0 beta

### Steps to Reproduce

1. unzip atom window zip (or the 64bit package, doesn't matter)
2. create .atom folder in parallel to the Atom 'installation folder' OR define ATOM_HOME to point to a '.atom' folder
3. startup atom with --portable flag
4. .atom is created and used in the user home and the requested portable .atom folder is ignored.

**Expected behavior:**

Portable mode should work same as in 1.13

**Actual behavior:** 

.atom folder is created and used from user home dir

**Reproduces how often:**

always

### Versions

1.14.0 beta from the releases tab in the github project site

**Comment by damieng** [comment]
Thanks for reporting this - looks like it gets incorrectly calculated during the mac/linux portable work and is expecting to find it in the same directory as the atom.exe right now. 

Fix it checked in and will be included in the next beta update. https://github.com/atom/atom/pull/13605/commits/9648d8b82fdb06cfaf3f8dcc14e910d06c742eb1

**Comment by sagiegurari** [comment]
Thanks 👍

**Comment by damieng** [comment]
Fix is included in 1.14.0-beta1 which will be out in a few minutes.

**Comment by sagiegurari** [comment]
doesn't work and seems like your PR fails on tests and wasn't merged into the master.

**Comment by damieng** [comment]
Yes, sorry it didn't make it after all.  It is now available on master - builds are available via appveyor if you want them ahead of the next beta update.

**Comment by sagiegurari** [comment]
thanks again. I'll wait for the official release on github

**Comment by sagiegurari** [comment]
still not working in 1.14.2 as well.

**Comment by damieng** [comment]
It was fixed in master but hadn't made it to the 1.14 branch. I just cherry-picked it over, it will be in 1.14.0-beta3.

**Comment by sagiegurari** [comment]
thanks
